spec/examples/error/mix.log

```diff
@@ -0,0 +1,6 @@
+Compiling 1 file (.ex)
+
+== Compilation error on file lib/error.ex ==
+** (TokenMissingError) lib/error.ex:5: missing terminator: end (for "do" starting at line 1)
+    (elixir) lib/kernel/parallel_compiler.ex:117: anonymous fn/4 in Kernel.ParallelCompiler.spawn_compilers/1
+
```

Comment:
The .js specs just use these `mix.log` files. The specs don't require Elixir. The Elixir code is just included for reference.